src/transformStateWithClones.js

```diff
@@ -64,8 +64,35 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const statesArr = [];
+  let stateCopObj = Object.assign({}, state);
+
+  transforms.forEach((operationObj) => {
+    if (operationObj.operation === 'addProperties') {
+      for (const key in operationObj.properties) {
+        stateCopObj[key] = operationObj.properties[key];
+      }
```

Comment:
```suggestion
      stateCopObj = {
        ...stateCopObj, ...properties,
      };
```
Можно использовать спред оператор для того что б добавить свойства.